### PR TITLE
Link for Connecticut added

### PR DESCRIPTION
**Pull request by hendrikebbers**

### CODE REVIEW

File: content/blog/adoptopenjdk-virtual-roadshow/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ AdoptOpenJDK is the leading provider of OpenJDK™ binaries. With over 170 milli
 |---|---|---|
 | 21/07/ 2020 18:30 EDT | Knoxville JUG  |  https://www.meetup.com/de-DE/KnoxJava/events/nmfmbrybckbcc/ |
 | 06/08/2020  19:00 GMT+3 | JUG Istanbul |  https://www.meetup.com/de-DE/Istanbul-Java-User-Group/events/271767087 |  
-| 11/08/2020  11:00 EDT | Connecticut JUG  | tbd |
+| 11/08/2020  11:00 EDT | Connecticut JUG  | tbd | https://www.meetup.com/de-DE/Connecticut-Java-Users-Group/events/271934054/
```
Comment (aahlenst) [inline content/blog/adoptopenjdk-virtual-roadshow/index.md:33]: You link to `de-DE`. https://www.meetup.com/Connecticut-Java-Users-Group/events/271934054/ might be more appropriate? Applies to all the other links to meetup.com, too.

File: content/blog/adoptopenjdk-virtual-roadshow/index.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ AdoptOpenJDK is the leading provider of OpenJDK™ binaries. With over 170 milli
 |---|---|---|
 | 21/07/ 2020 18:30 EDT | Knoxville JUG  |  https://www.meetup.com/de-DE/KnoxJava/events/nmfmbrybckbcc/ |
```
Comment (hendrikebbers) [inline content/blog/adoptopenjdk-virtual-roadshow/index.md:31]:
```suggestion
| 21/07/ 2020 18:30 EDT | Knoxville JUG  |  https://www.meetup.com/KnoxJava/events/nmfmbrybckbcc/ |
```

File: content/blog/adoptopenjdk-virtual-roadshow/index.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ AdoptOpenJDK is the leading provider of OpenJDK™ binaries. With over 170 milli
...
 | 06/08/2020  19:00 GMT+3 | JUG Istanbul |  https://www.meetup.com/de-DE/Istanbul-Java-User-Group/events/271767087 |
```
Comment (hendrikebbers) [inline content/blog/adoptopenjdk-virtual-roadshow/index.md:32]:
```suggestion
| 06/08/2020  19:00 GMT+3 | JUG Istanbul |  https://www.meetup.com/Istanbul-Java-User-Group/events/271767087 |  
```

Comment (hendrikebbers) [reply content/blog/adoptopenjdk-virtual-roadshow/index.md:32]:
```suggestion
| 06/08/2020  19:00 GMT+3 | JUG Istanbul |  https://www.meetup.com/Istanbul-Java-User-Group/events/271767087 |  
```

File: content/blog/adoptopenjdk-virtual-roadshow/index.md (hunk 1 above)
Comment (hendrikebbers) [reply content/blog/adoptopenjdk-virtual-roadshow/index.md:33]:
```suggestion
| 11/08/2020  11:00 EDT | Connecticut JUG  | tbd | https://www.meetup.com/Connecticut-Java-Users-Group/events/271934054/
```

File: content/blog/adoptopenjdk-virtual-roadshow/index.md [hunk 4]
```diff
@@ -30,7 +30,7 @@ AdoptOpenJDK is the leading provider of OpenJDK™ binaries. With over 170 milli
 |---|---|---|
 | 21/07/ 2020 18:30 EDT | Knoxville JUG  |  https://www.meetup.com/de-DE/KnoxJava/events/nmfmbrybckbcc/ |
 | 06/08/2020  19:00 GMT+3 | JUG Istanbul |  https://www.meetup.com/de-DE/Istanbul-Java-User-Group/events/271767087 |  
-| 11/08/2020  11:00 EDT | Connecticut JUG  | tbd |
+| 11/08/2020  11:00 EDT | Connecticut JUG  | tbd | https://www.meetup.com/de-DE/Connecticut-Java-Users-Group/events/271934054/
 | 12/08/2020 | JUG Switzerland  | tbd |
 | 18/08/2020 | JUG Dortmund  | https://www.meetup.com/de-DE/JUG-Dortmund/events/272181147/ |
```
Comment (hendrikebbers) [inline content/blog/adoptopenjdk-virtual-roadshow/index.md:35]:
```suggestion
| 18/08/2020 | JUG Dortmund  | https://www.meetup.com/JUG-Dortmund/events/272181147/ |
```

**Pull request merged**